Documentation/trying-out-rkt.md

```diff
@@ -1,13 +1,14 @@
-# Trying out rkt 
+# Trying out rkt
 
-This guide provides a short introduction to trying out rkt.
-For a more in-depth guide of a full end-to-end workflow of building an application and running it using rkt, check out the [getting-started-guide](getting-started-guide.md)
+This document introduces the basics of trying out rkt. For a more in-depth guide to building an application container and running it with rkt, check out the [getting started guide][getstart].
 
 ## Using rkt on Linux
 
-rkt consists of a single CLI tool and can run on different platforms. The primary target platform currently is amd64 Linux. A modern kernel is required but there should be no other system dependencies.
+rkt consists of a single command line program compiled for Linux on the amd64 architecture. A modern Linux kernel is required, but there are no other system prerequisites.
```

Comment:
I think this is going in the exact opposite direction of https://github.com/coreos/rkt/pull/2808#discussion_r67760117 Or is that on purpose and I'm lacking some further context here?